barretenberg/cpp/src/barretenberg/vm2/constraining/recursion/recursive_verifier.test.cpp

```diff
@@ -164,13 +162,9 @@ TEST_F(AvmRecursiveTests, GoblinRecursion)
     // Type aliases specific to GoblinRecursion test
     using AvmRecursiveVerifier = AvmGoblinRecursiveVerifier;
     using UltraRollupRecursiveFlavor = UltraRollupRecursiveFlavor_<UltraRollupFlavor::CircuitBuilder>;
-    using Curve = UltraRollupRecursiveFlavor::Curve;
     using UltraFF = UltraRollupRecursiveFlavor::FF;
     using UltraRollupProver = UltraProver_<UltraRollupFlavor>;
-
-    if (testing::skip_slow_tests()) {
-        GTEST_SKIP();
```

Comment:
Was merging this intended? No need to stop your PR even if it wasnt! Just pointing out so that we don't forget.

(locally you can run the test by setting AVM_SLOW_TESTS=1 [I think that was the env variable, it's in the fixtures file]. Jean has an task to make this run in CI_FULL)

Reply:
Yes, this was indeed intentional. This test is super high value right now and is actually not very expensive so unless you guys have a particular reason to avoid running it I'd like to keep it active by default!